data_processing_module3.py

```diff
@@ -0,0 +1,28 @@
+import numpy as np
+from sklearn import preprocessing
+from sklearn.cross_validation import train_test_split
+import random
+
+X1=np.asarray([random.sample(xrange(1000), 10), random.sample(xrange(1000), 10), random.sample(xrange(1000), 10), random.sample(xrange(1000), 10), random.sample(xrange(1000), 10), random.sample(xrange(1000), 10), random.sample(xrange(1000), 10), random.sample(xrange(1000), 10), random.sample(xrange(1000), 10)], dtype=np.float)
+X2=np.asarray([random.sample(xrange(1000), 10), random.sample(xrange(1000), 10), random.sample(xrange(1000), 10), random.sample(xrange(1000), 10), random.sample(xrange(1000), 10), random.sample(xrange(1000), 10), random.sample(xrange(1000), 10), random.sample(xrange(1000), 10), random.sample(xrange(1000), 10)], dtype=np.float)
+X3=np.asarray([random.sample(xrange(1000), 10), random.sample(xrange(1000), 10), random.sample(xrange(1000), 10), random.sample(xrange(1000), 10), random.sample(xrange(1000), 10), random.sample(xrange(1000), 10), random.sample(xrange(1000), 10), random.sample(xrange(1000), 10), random.sample(xrange(1000), 10)], dtype=np.float)
+Y=np.asarray([random.sample(xrange(1000), 10), random.sample(xrange(1000), 10), random.sample(xrange(1000), 10), random.sample(xrange(1000), 10), random.sample(xrange(1000), 10), random.sample(xrange(1000), 10), random.sample(xrange(1000), 10), random.sample(xrange(1000), 10), random.sample(xrange(1000), 10)], dtype=np.float)
+W=np.asarray([random.sample(xrange(1000), 10), random.sample(xrange(1000), 10), random.sample(xrange(1000), 10), random.sample(xrange(1000), 10), random.sample(xrange(1000), 10), random.sample(xrange(1000), 10), random.sample(xrange(1000), 10), random.sample(xrange(1000), 10), random.sample(xrange(1000), 10)], dtype=np.float)
```

Comment:
Lines 6-10 are not part of a function and I assumed they were part of your testing module. You can feel free to remove them from here. Also, by the way, to create an NxM array of random numbers, you can simply use `np.random.random((N,M))`